java/code/src/com/redhat/rhn/manager/kickstart/cobbler/test/CobblerEnableBootstrapCommandTest.java

```diff
@@ -50,8 +50,8 @@ public class CobblerEnableBootstrapCommandTest extends BaseTestCaseWithUser {
 
     @Override
     @BeforeEach
```

Comment:
This change is needed too

```suggestion
    @AfterEach
```